heroku/cli
# refactor(spaces): Move command spaces:rename to oclif

### Description
https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00001b1T43YAE/view

This converts `spaces:rename` to oclif and its associated tests.

### Testing
1) Have a test space (do not rename any of our production spaces, please)
2) Pull down this branch and run `yarn build`
3) Run `./bin/run spaces:rename --from SPACE_NAME --to NEW_SPACE_NAME` on a space
5) Verify the command succeeds and the space is renamed (`heroku spaces:info NEW_SPACE_NAME`)
6) Compare with prod
